From: Arnaud Ferraris <arnaud.ferraris@gmail.com>
Date: Sat, 20 Nov 2021 12:51:33 +0100
Subject: pine64_defconfig: compile typec support built-in

Recent patches made the PinePhone's HDMI output rely on `anx7688` acting
as extcon, meaning we must now have this driver built-in instead of as a
module.

defconfig re-generation also caused an impactless change in the DRM
config.
---
 arch/arm64/configs/pine64_defconfig | 17 +++++++----------
 1 file changed, 7 insertions(+), 10 deletions(-)

diff --git a/arch/arm64/configs/pine64_defconfig b/arch/arm64/configs/pine64_defconfig
index d8cdb8b..6d9c5e9 100644
--- a/arch/arm64/configs/pine64_defconfig
+++ b/arch/arm64/configs/pine64_defconfig
@@ -1399,12 +1399,12 @@ CONFIG_DRM_PANEL_SITRONIX_ST7703=y
 CONFIG_DRM_DW_HDMI_AHB_AUDIO=m
 CONFIG_DRM_DW_HDMI_I2S_AUDIO=m
 CONFIG_DRM_DW_HDMI_CEC=m
+CONFIG_DRM_SIMPLEDRM=y
 CONFIG_DRM_LIMA=y
 CONFIG_FB=y
 CONFIG_FIRMWARE_EDID=y
 CONFIG_FB_SMSCUFX=m
 CONFIG_FB_UDL=m
-CONFIG_FB_SIMPLE=y
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_BACKLIGHT_PWM=y
 CONFIG_FRAMEBUFFER_CONSOLE=y
@@ -1704,15 +1704,12 @@ CONFIG_USB_G_MULTI=m
 CONFIG_USB_G_MULTI_CDC=y
 CONFIG_USB_G_HID=m
 CONFIG_USB_G_DBGP=m
-CONFIG_TYPEC=m
-CONFIG_TYPEC_TCPM=m
-CONFIG_TYPEC_TCPCI=m
-CONFIG_TYPEC_FUSB302=m
-CONFIG_TYPEC_UCSI=m
-CONFIG_TYPEC_ANX7688=m
-CONFIG_TYPEC_HD3SS3220=m
-CONFIG_TYPEC_DP_ALTMODE=m
-CONFIG_TYPEC_NVIDIA_ALTMODE=m
+CONFIG_TYPEC=y
+CONFIG_TYPEC_TCPM=y
+CONFIG_TYPEC_TCPCI=y
+CONFIG_TYPEC_UCSI=y
+CONFIG_TYPEC_ANX7688=y
+CONFIG_TYPEC_DP_ALTMODE=y
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=256
 CONFIG_SDIO_UART=m
